selectors.yml

```diff
@@ -1,11 +1,17 @@
 selectors:
   # Testnets
+  31: 
+    selector: 8953668971247136127 
+    name: "bitcoin-testnet-rootstock"
```

Comment:
Is this evm?

Reply:
they have their own VM that is compatible with EVM